test/test_data_type.py

```diff
@@ -48,6 +48,10 @@ def test_datatype_f64(self):
     else:
       assert 'f64' in device_data_hlo, device_data_hlo
 
+  @unittest.skipIf(
+      os.environ.get('XLA_USE_EAGER_DEBUG_MODE'),
```

Comment:
I wonder what this flag means. iow, why do we need to disable this test (i think it's failing on TPU VM now) when this flag is set.

Reply:
> I wonder what this flag means. iow, why do we need to disable this test (i think it's failing on TPU VM now) when this flag is set.

Thanks for the reminder!
When I tested locally, I use this flag to disable failed tests, and will replace it with another flag when all the failed tests disabled